ios/config.json

```diff
@@ -0,0 +1,29 @@
+{
+    "code": [
```

Comment:
this project seems to be using 2 line spacing, this particular file is 4 lines spacing